www/src/components/docs-breadcrumb.js

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import { Link } from "gatsby"
+import getActiveItem from "../utils/sidebar/get-active-item"
+import getActiveItemParents from "../utils/sidebar/get-active-item-parents"
+import { mediaQueries, space, fontSizes } from "../utils/presets"
+import { rhythm } from "../utils/typography"
+
+const Separator = ({ character = `>` }) => (
+  <span style={{ margin: `0px ${space[1]}` }} role="presentation">
+    {character}
+  </span>
+)
+
+const BreadcrumbNav = ({ children, mobile = false }) => (
+  <nav
+    aria-label="breadcrumb"
+    css={{
+      fontSize: fontSizes[1],
+      display: `${mobile ? `inherit` : `none`}`,
+      [mediaQueries.md]: {
+        display: `${mobile ? `none` : `inherit`}`,
+      },
+      marginBottom: rhythm(1 / 2),
+    }}
+  >
+    {children}
+  </nav>
+)
+
+const Breadcrumb = ({ itemList, location }) => {
+  const activeItem = getActiveItem(itemList.items, location, undefined)
+  const activeItemParents = getActiveItemParents(itemList.items, activeItem, [])
+
+  // return a shorter version of the breadcrumb on the intro page
+  // because the docs intro page isn't generated from markdown
+  if (activeItem.title === `Introduction`) {
+    return (
+      <>
+        {/* only the breadcrumb nav of the proper viewport is displayed */}
+        <BreadcrumbNav>
+          <Link to="/">Home</Link>
+          <Separator />
+          Docs
+        </BreadcrumbNav>
+        <BreadcrumbNav mobile>
+          <Separator character="<" />
+          <Link to="/">Home</Link>
+        </BreadcrumbNav>
+      </>
+    )
+  }
+
+  return (
+    <>
+      <BreadcrumbNav>
+        <Link to="/">Home</Link>
+        <Separator />
+        <Link to="/docs/">Docs</Link>
+        <Separator />
+        {activeItemParents.reverse().map(item => (
+          <React.Fragment key={item.title}>
+            <span>
+              <Link to={item.link}>{item.title}</Link>
+            </span>
+            <Separator />
+          </React.Fragment>
+        ))}
+        {activeItem.title}
```

Comment:
```suggestion
        <span aria-current="page">{activeItem.title}</span>
```
I think we should mark the current page somehow. `aria-current` is the recommended way to do that for assistive tech, and after consulting the [ARIA spec](https://www.w3.org/TR/wai-aria-1.1/#aria-current) it looks like it's allowed on any element.

Reply:
I love getting this kind of feedback! Thanks for catching!